### PR TITLE
[GCOS] Add GCOS-specific mnemonics TERMINAL, ALTERNATE-CONSOLE, ALTERNATE CONSOLE

### PR DESCRIPTION
**Pull request by ddeclerck**

### CODE REVIEW

File: config/gcos-strict.conf [hunk 1]
```diff
@@ -39,6 +41,9 @@ pretty-display:			yes
 # Allow complex OCCURS DEPENDING ON
 complex-odo:			no
 
+# Adjust position of items following OCCURS DEPENDING
```
Comment (nberth) [inline config/gcos-strict.conf:44]: I think these should belong to another pull request ;-)

File: config/gcos-strict.conf [hunk 2]
```diff
@@ -231,6 +236,7 @@ records-mismatch-record-clause:		error
 record-delimiter:			ok
 sequential-delimiters:			ok
 record-delim-with-fixed-recs:		unconformable
+record-sequential-advancing:		ok
```
Comment (nberth) [inline config/gcos-strict.conf:239]: As above.

File: config/gcos-strict.conf (hunk 1 above)
Comment (lefessan) [reply config/gcos-strict.conf:44]: To be honest, it's very likely that will be squashing all these PRs into a single commit ("add GCOS dialect") sent as a single patch to the official GNUCobol team. So, no need to be extract careful about splitting PRs and commits for now. We may split it at some point to extract things that are not specifically tied to GCOS.
